creusot-contracts/src/ghost_ptr.rs

```diff
@@ -257,7 +271,7 @@ impl<'a, T> DerefMut for GhostPtrTokenMut<'a, T> {
 
 #[trusted]
 impl<'a, T> Resolve for GhostPtrTokenMut<'a, T> {
-    #[predicate]
+    #[predicate(prophetic)]
```

Comment:
Curious... this wasn't causing an error in CI? I thought we ran `cargo creusot` on `creusot-contracts` in CI?

Reply:
The resolve didn't need to be prophetic when `self.fin()` wasn't prophetic, but when I made `GhostPtrToken::fin` prophetic, `cargo creusot` broken until I also fixed `resolve`.